src/store/appModes/reducers.ts

```diff
@@ -0,0 +1,29 @@
+import { MODIFY_APP_MODES } from './actions';
+
+export interface AppModeReducer {
+  isExtensionDisabled: boolean;
+  annotationModePages: string[];
+  disabledPages: string[];
+}
+
+const initialState: AppModeReducer = {
+  isExtensionDisabled: false,
+  annotationModePages: [],
+  disabledPages: [],
+};
+
+export default function appModes(state = initialState, action) {
+  switch (action.type) {
+    case MODIFY_APP_MODES:
+      return appModesActionHandler(state, action.payload);
```

Comment:
Calling this trivial function does not make it more descriptive to me and it definitely gets harder to read with every action being trivial function that I have to follow just to find it's trivial. Do we need this layer? Why not using `{ ...state,  ...payload,  }` directly?

Reply:
Usually action handlers are split into functions if they are a bit more complex (and if there's a lot of them), but in this case I'd say we can do it like @glowka suggested, especially that it's rather very simple.

Unless we want to have a strict style of writing reducers (maybe better for general code readability - same solutions used everywhere, one decision less to make when writing new code)